nestkernel/node.h

```diff
@@ -477,6 +477,59 @@ class Node
   virtual void handle( DoubleDataEvent& e );
 
   /**
+   * return the Ca_minus value at t (in ms).
+   * return 0.0 if not overridden
+   */
+  virtual double_t
+  get_Ca_value() const
+  {
+    return 0.0;
+  }
+
+  /**
+   * get the number of synaptic element for the current Node at t (in ms)
+   * return 0.0 if not overridden
+   */
+  virtual double_t get_synaptic_element( Name ) const
```

Comment:
It appears that this method would return an integer. Why is the return type double?